README.rst

```diff
@@ -1,3 +1,23 @@
+.. image:: https://img.shields.io/pypi/v/waitress.svg
+    :target: https://pypi.python.org/pypi/waitress
+    :alt: waitress on PyPI
```

Comment:
I feel that a screen reader user needs to know where the link is headed, which is the PyPi on waitress. It's unfortunate that we cannot load the version number and put it automatically in the alt attribute.